src/devices/cpu/z80/z80.lst

```diff
@@ -6763,9 +6359,9 @@ edff	# DB   ED
 
 00db	# IN   A,(n)
 	call arg
-	TADR = TDAT8 | (A << 8);
-	call in
-	A = TDAT8; WZ = TADR + 1;
+	TDAT2 = TDAT8 | (A << 8);
+	call in TDAT2
+	A = TDAT8; WZ = (TDAT2 | (A << 8)) + 1;
```

Comment:
OK tell me what I’m missing.  To me, this looks like:
* Get n in TDAT8
* Store full 16-bit address `(A << 8) | TDAT8` in TDAT2
* Do I/O read into TDAT8
* Store value read into A
* Update WZ with full address OR'd with new A value plus one

The old code doesn’t appear to OR with the new A value.  I would’ve expected:
```
00db	# IN   A,(n)
	call arg
	TADR = TDAT8 | (A << 8);
	call in
	A = TDAT8; WZ = TADR + 1;
	TDAT2 = TDAT8 | (A << 8);
	call in TDAT2
	A = TDAT8; WZ = TDAT2 + 1;
```

Reply:
Sorry, copy/paste error, updated comment to include snippet:
```
00db	# IN   A,(n)
	call arg
	TDAT2 = TDAT8 | (A << 8);
	call in TDAT2
	A = TDAT8; WZ = TDAT2 + 1;
```

Reply:
Sorry for been inattentive. You're right and thank you for been such!